launch_ros/launch_ros/actions/load_composable_nodes.py

```diff
@@ -281,6 +283,11 @@ def get_composable_node_load_request(
             if isinstance(param, tuple):
                 subs = normalize_parameter_dict({param[0]: param[1]})
                 parameters.append(subs)
+            else:
+                param_file_path = os.path.abspath(param)
+                assert os.path.isfile(param_file_path)
+                subs = ParameterFile(param_file_path)
```

Comment:
nit: prefer using `pathlib`

```suggestion
                param_file_path = Path(param).resolve()
                assert param_file_path.is_file()
                subs = ParameterFile(param_file_path)
```

you will need `from pathlib import Path` at the beggining of the file.